arkane/statmech.py

```diff
@@ -788,6 +788,10 @@ def projectRotors(conformer, F, rotors, linear, is_ts, label):
     Refer to Gaussian whitepaper (http://gaussian.com/vib/) for procedure to calculate
     harmonic oscillator vibrational frequencies using the force constant matrix.
     """
+
+    Nrotors = sum([2 if len(x)==8 else 1 for x in rotors])
```

Comment:
The problem is that the ProjectRotors function doesn't take rotor objects so that information isn't readily available, (otherwise I would use rotor.dof for the new ones).